frontend/express/public/stylesheets/styles/blocks/_dropdown.scss

```diff
@@ -0,0 +1,79 @@
+@use "../base/typography-variables" as t;
+
+.cly-vue-menubox-new {
+    &.menubox-default-skin {
+        display: inline-block;
+        .el-input__inner::placeholder{
+            color:black;
+        }
+
+        .menu-body {
+            box-shadow: 0 3px 7px rgba(0, 0, 0, 0.08);
+            background-color: #fff;
+            position: absolute;
+            top: 0px;
+            z-index: 100;
+            border: 1px solid #d0d0d0;
+            border-radius: 2px;
+            padding: 16px;
+            width: max-content;
+            .controls {
+                float: right;
+                margin-top: 10px;
+            }
+        }
+
+    }
+    &__title{
+        color: #333C48;
+        font-family:  t.$font-family;
+        line-height: 24px;
+        font-weight: 500;
+        font-size: 16px;
+	  }
+	  &__reset{
+		float: right;
+		font-size: 12px;
+		padding-bottom: 26px;
+		color: #D23F00;
+		cursor: pointer;
+        font-family:  t.$font-family;
+        line-height: 20px;
+        font-weight: 500;
+	  }
+      &__field{
+        color: #333C48;
+        font-family:  t.$font-family;
+        line-height: 20px;
+        font-weight: 500;
+        font-size: 12px;
+        padding-bottom: 3px;
+	  }
+      &__field-dropdown{
+        width:320px;
+        height:32px;
+        border-radius:4px;
+        border:1px solid #E2E4E8;
+        margin-bottom:16px;
+	  }
+      &__confirm{
+		background-color: #12AF51;
+		color: white;
+        font-family: t.$font-family;
+        font-weight: 500;
+        font-size: 13px;
+        line-height: 13px;
+	  }
+      &__cancel {
+        width:67px;
+        height:32px;
+        border-radius: 4px;
+        background: #F6F6F6;
+        padding: 10px 12px;
+      }
+      &__dropdown {
+        .el-input__inner::placeholder{
+            color:black;
+        }
+      }
```

Comment:
These are probably some debris from the legacy component I wrote, but you should definitely avoid this much of custom styling for fundamental elements. For instance, buttons shouldn't be styled here, font-family shouldn't be specified here. The bottom line is that custom styling should be the last resort.